README.md

```diff
@@ -12,15 +11,14 @@ to get further details about procedures and any other  to get started.
+This project is a Jekyll site that uses a [remote theme](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll) from [Just the Docs](https://pmarsceill.github.io/just-the-docs/), and is hosted on [GitHub Pages](https://pages.github.com/). The pages are primarily developed in Markdown and proceessed via [kramdown](https://kramdown.gettalong.org/). Customization to the themes and layouts are done with [SCSS](https://sass-lang.com/documentation/syntax#scss) and HTML. Most of the high level configuration can be found in [`_config.yml`](_config.yml) and is a mix of configuration from Jekyll and Just the Docs. To learn more about how to create and organize content refer to [Jekyll Docs](https://jekyllrb.com/docs/pages/). To customize the layout and theme, refer to the appropriate section under [Just the Docs - Customization](https://pmarsceill.github.io/just-the-docs/docs/customization/).
 
 ## Testing Locally
-To test locally you can use serve up the files using
+Test the site locally by running the following command:
 ```
 $ bundle exec jekyll serve --watch
 ```
-If the server is running you can visit http://localhost:4000 to see what it looks like. With the `--watch` flag you can make changes to the content and
-it will live reload the changes, without having to stop the server.
+Once the server is up and running, visit http://localhost:4000 in a browser to validate the layout and content. The `--watch` flag allows you to change  content without having to stop and restart the server (live reload).
 
 ## Contributing
-If you observe an issue on [easydynamics.github.io](https://easydynamics.github.io/), see if it's related to any of the [existing ones](https://github.com/EasyDynamics/easydynamics.github.io/issues) and join the conversation. If not, open a new issue and describe it for us. If you want to submit a change, open an issue first and then submit a pull request associated with the issue accordingly. Last but not least please follow [GitHub Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).
\ No newline at end of file
+If you notice a bug or want to report an issue with [easydynamics.github.io](https://easydynamics.github.io/), see if it's related to any of the [existing ones](https://github.com/EasyDynamics/easydynamics.github.io/issues) and join the conversation. If not, open a new issue and describe it for us. If you want to fix minor bugs or issues, feel free to open a pull request with the appropriate changes. For broader topics, start a conversation on an issue and build consensus with the maintainers before creating a pull request. Last but not least please follow [GitHub Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).
```

Comment:
```suggestion
If you notice a bug or want to report an issue with [easydynamics.github.io](https://easydynamics.github.io/), see if it's related to any of the [existing ones](https://github.com/EasyDynamics/easydynamics.github.io/issues) and join the conversation. If not, open a new issue and describe it for us. If you want to fix minor bugs or issues, feel free to open a pull request with the appropriate changes. For broader topics, start a conversation on an issue and build consensus with the maintainers before creating a pull request. Last but not least, please follow [GitHub Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).
```

Reply:
Accidentally hit commit suggestion instead of adding it to the batch. Will fix this comma on a another commit.